udata/search/fields.py

```diff
@@ -56,7 +58,20 @@ def get_value_filter(self, value):
 
 
 class TermsFacet(Facet, DSLTermsFacet):
-    pass
+
+    def add_filter(self, filter_values):
+        if not filter_values:
```

Comment:
Actually, if you plan to make a PR against this one, why not merge that one directly and then improve?